src/entity/Action.ts

```diff
@@ -0,0 +1,32 @@
+import { Session } from './Session';
+import { Command } from './Command';
+import { Exception } from './Exception';
+
+import {
+	Column,
+	Entity,
+	ManyToOne,
+	OneToMany,
+	PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity()
+export class Action {
+	@PrimaryGeneratedColumn()
+	id: number;
+
+	@ManyToOne(() => Session, session => session.actions)
+	session: Session;
+
+	@OneToMany(() => Command, command => command.action)
+	commands: Command[];
+
+	@OneToMany(() => Exception, exception => exception.action)
+	exceptions: Exception[];
+
+	@Column()
+	created_at: Date;
```

Comment:
Might be nice to rename this to reported_at if we rename the Action model to Event